src/Http/Middleware/IdpMiddleware.php

```diff
@@ -93,22 +95,19 @@ public function handle(Request $request, Closure $next)
      */
     protected function retrievePermissions($userId, array $roles)
     {
-        $permissions = [];
-        $grant = new Grant();
-        if (config('idp.connection') === 'mongodb') {
```

Comment:
Siamo sicuri di poterlo togliere?

Reply:
Sì, dalle informazioni che abbiamo raccolto veniva usato in una versione precedente di Sendy, se non erro. E non ci risultano altri utilizzi.